lib/routes/cfr/utils.ts

```diff
@@ -0,0 +1,251 @@
+    if ($articleHeader.find('.article-header__image-img').length) {
+        const articleImageSrc = $articleHeader.find('.article-header__image-img').attr('src');
+        description = `<img src="${articleImageSrc}"><br>${description}`;
+    }
+    const videoIfame = getVideoIframe($articleHeader);
+    if (videoIfame) {
+        description = `${videoIfame}<br>${description}`;
+    }
+    return {
+        title: linkData?.title ?? $('.article-header__title').text(),
+        pubDate: linkData?.pubDate,
+        description,
+    };
+}
+
+function parseBlog($: CheerioAPI): DataItem {
+    const linkData = parseLinkData($);
+    const description = parseDescription($('.body-content'), $);
+    return {
+        title: linkData?.title ?? $('.article-header-blog__title').text(),
+        pubDate: linkData?.pubDate,
+        description,
+    };
+}
+
+function parseBook($: CheerioAPI): DataItem {
+    const linkData = parseLinkData($);
+    let description = parseDescription($('.body-content'), $);
+    const sectionTop = $('.article-header__section-top').remove('.article-header__image');
+    description = `${sectionTop.html()}<br>${description}`;
+    const bookCover = $('.article-header__image-img').attr('src');
+    if (bookCover) {
+        description = `<img src="${bookCover}"><br>${description}`;
+    }
+
+    return {
+        title: linkData?.title ?? $('.article-header__title').text(),
+        pubDate: linkData?.pubDate,
+        description,
+    };
+}
+
+function parseConferenceCalls($: CheerioAPI): DataItem {
+    const linkData = parseLinkData($);
+    const description = parseDescription($('.podcast-body').last(), $);
+    return {
+        title: linkData?.title ?? $('head title').text(),
+        pubDate: linkData?.pubDate,
+        description,
+    };
+}
+
+function parseEvent($: CheerioAPI): DataItem {
+    const linkData = parseLinkData($);
+    let description = parseDescription($('.body-content'), $);
+    const videoIfame = getVideoIframe($('.msp-event-video'));
+    if (videoIfame) {
+        description = `${videoIfame}<br>${description}`;
+    }
+
+    return {
+        title: linkData?.title ?? $('.msp-event-header-past__title').text(),
+        pubDate: linkData?.pubDate,
+        description,
+    };
+}
+
+function parseBackgrounder($: CheerioAPI): DataItem {
+    const linkData = parseLinkData($);
+    let description = parseDescription($('.main-wrapper__article-body .body-content'), $);
+    const summary = $('.main-wrapper__article-body .summary').html();
+    if (summary) {
+        description = `${summary}<br>${description}`;
+    }
+
+    return {
+        title: linkData?.title ?? $('.article-header-backgrounder__title').text(),
+        pubDate: linkData?.pubDate,
+        description,
+    };
+}
+
+function parsePodcasts($: CheerioAPI): DataItem {
+    const linkData = parseLinkData($);
+    const description = $('.body-content').first().html() ?? '';
```

Comment:
Podcasts should have their audio embeded in either `description` or [`enclosure`](https://docs.rsshub.app/joinus/advanced/advanced-feed#create-a-podcast-feed)